submit_cgap/submission.py

```diff
@@ -666,20 +658,49 @@ def show_upload_info(uuid, server=None, env=None, keydict=None):
     :param keydict: keydict-style auth, a dictionary of 'key', 'secret', and 'server'
     """
 
+    if app is None:
+        app = DEFAULT_APP
+    if KEY_MANAGER.selected_app != app:
+        with KEY_MANAGER.locally_selected_app(app):
+            return show_upload_info(uuid=uuid, server=server, env=env, keydict=keydict, app=app)
+
     server = resolve_server(server=server, env=env)
     keydict = keydict or KEY_MANAGER.get_keydict_for_server(server)
     url = ingestion_submission_item_url(server, uuid)
-    response = requests.get(url, auth=KEY_MANAGER.keydict_to_keypair(keydict))
+    response = portal_request_get(url, auth=KEY_MANAGER.keydict_to_keypair(keydict), headers=STANDARD_HTTP_HEADERS)
     response.raise_for_status()
     res = response.json()
     if get_section(res, 'upload_info'):
         show_section(res, 'upload_info')
     else:
-        show("No uploads.")
-
-
-def do_any_uploads(res, keydict, upload_folder=None, ingestion_filename=None,
-                   no_query=False, subfolders=False):
+        show("Uploads: None")
+
+    # New March 2023 ...
+
+    if get_section(res, 'validation_output'):
```

Comment:
Should these be conditional on arguments like:
```
def show_upload_info(uuid, server=None, env=None, keydict=None, app: str = None,
                     show_upload_info=True, show_validation_output=True,
                     show_processing_status=True, show_parameters=True):
```
so that a caller could select just one of these if that's all they wanted? Then
```
    if show_processing_status and res.get('proessing_status'):
```
etc.

Not sure parameters should be shown last. Maybe first? Parameters are what decide whether there _is_ validation output, for example.